[2021-04-22 02:00:12,539] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop 2021-04-21T00:00:00+00:00 [queued]>
[2021-04-22 02:00:12,545] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop 2021-04-21T00:00:00+00:00 [queued]>
[2021-04-22 02:00:12,546] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-04-22 02:00:12,546] {taskinstance.py:1069} INFO - Starting attempt 1 of 1
[2021-04-22 02:00:12,546] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-04-22 02:00:12,551] {taskinstance.py:1089} INFO - Executing <Task(BashOperator): run_after_loop> on 2021-04-21T00:00:00+00:00
[2021-04-22 02:00:12,553] {standard_task_runner.py:52} INFO - Started process 55582 to run task
[2021-04-22 02:00:12,560] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'run_after_loop', '2021-04-21T00:00:00+00:00', '--job-id', '17', '--pool', 'default_pool', '--raw', '--subdir', '/Users/wschrep/FreelanceWork/VIAA/IIIF_newproject/python_env/lib/python3.8/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/var/folders/68/c_pd8_vj55d9mdvv3mvllkh00000gn/T/tmp5sogd89g', '--error-file', '/var/folders/68/c_pd8_vj55d9mdvv3mvllkh00000gn/T/tmp1ng1yoja']
[2021-04-22 02:00:12,562] {standard_task_runner.py:77} INFO - Job 17: Subtask run_after_loop
[2021-04-22 02:00:12,594] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.run_after_loop 2021-04-21T00:00:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2021-04-22 02:00:12,621] {taskinstance.py:1281} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=run_after_loop
AIRFLOW_CTX_EXECUTION_DATE=2021-04-21T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-21T00:00:00+00:00
[2021-04-22 02:00:12,621] {bash.py:135} INFO - Tmp dir root location: 
 /var/folders/68/c_pd8_vj55d9mdvv3mvllkh00000gn/T
[2021-04-22 02:00:12,622] {bash.py:158} INFO - Running command: echo 1
[2021-04-22 02:00:12,629] {bash.py:169} INFO - Output:
[2021-04-22 02:00:12,633] {bash.py:173} INFO - 1
[2021-04-22 02:00:12,634] {bash.py:177} INFO - Command exited with return code 0
[2021-04-22 02:00:12,651] {taskinstance.py:1185} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=run_after_loop, execution_date=20210421T000000, start_date=20210422T000012, end_date=20210422T000012
[2021-04-22 02:00:12,672] {taskinstance.py:1246} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-04-22 02:00:12,720] {local_task_job.py:146} INFO - Task exited with return code 0
